src/components/ListingCard/index.tsx

```diff
@@ -18,7 +17,7 @@ export default function ListingCard({
 }: {
   caseData: CaseListing;
   isSelected?: boolean;
-  onClick?: (id: UUID) => void;
+  onClick?: (id: string) => void;
 }) {
   // Helper function to generate an array of strings for CardTag components
   const generateCardTags = (): string[] => {
```

Comment:
looks like the bug/console error in the screenshot below was introduced because `caseData.relief_codes` and `caseData.languages` are both lists with one null entry. This results in empty tags on the `ListingCard` component, even when there are no languages or relief codes associated with the case listing. Could we try getting rid of these console log errors? If you want a quick solution we can update the `if` checks on lines 34 and 42 to check that the 0th of lists does not equal null (`caseData.languages[0] !== null`), but there might be a less hacky fix? <img width="1432" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/73e73f95-7244-48a1-ad9e-4bde9c215150">

Reply:
i believe that's out of scope for this sprint, but i can include that for now
it would be better if we do a full fix of the cases page as another sprint.

Reply:
THANK U FOR THE FIX!! But yeah, I agree that we'll need to make some fixes to the cases page in future sprints 👍 just wanted to avoid introducing console errors into the main branch